elizhahn/overlook
# Feature/dashboard

# Is this a fix or a feature?
- feature
# What is the change?
- markup and styling was added for the customer's dashboard page
- partial scss files were started with intention of DRY css
# What does it fix?
# Where should the reviewer start?
- index.html
- base.scss
# How should this be tested?
- open index.html on chrome, safari, firefox and all styling should be the same
- Styling should be appealing
- run an accessibility audit, audit should be at 100%
# Screenshots (if appropriate):
